liota/dcc/vrops.py

```diff
@@ -165,12 +164,13 @@ def properties(self, msg_id, res_uuid, res_kind, timestamp, key, value):
                 "property_data": []
             }
         }
-        msg["body"]["property_data"].append({"propertyKey": key, "propertyValue": value})
+        for property_dict in properties_list: 
+            msg["body"]["property_data"].append({"propertyKey": property_dict.keys()[0], "propertyValue":  property_dict.values()[0]})
```

Comment:
This looks proper. Just one suggestion in case if we have multiple items in the dictionary as user will be providing this data(we don`t have control), the safe code to iterate over dictionary using the following call;

for key, value in property_dict.iteritems():

rather then just referring the first element.